application-chartjs-ui/src/main/resources/XWiki/ChartJS/WebHome.xml

```diff
@@ -336,6 +337,14 @@ require(['jquery', 'chart', 'datalabels', 'palette'], function($, Chart, datalab
         data: data,
         options: options
       });
+
+      // If the table is hidden, move it to the canvas element and show it, for accessibility purpose.
```

Comment:
> and show it

This part is confusing. Why show the table if the user wanted it to be hidden? You probably meant:

> so that it can be read by screen readers

Reply:
Thank you Marius for the thorough review. I take good note of your advice to use one pull request per issue. Regarding your comment above: indeed, I meant "so that it can be read by screen readers", I updated the comment accordingly.